docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,739 @@
+
+We are happy to report that we have migrated **all** our test codebase to Testing Library.
+You can use our tests as inspiration when you need to write new ones.
+The migration has even allowed us to rethink some of the implementations, making the components easier to test with the library.
+
+### TypeScript migration
+
+The MUI Core codebase is not completely written in TypeScript yet but we have been coming a long way ([MUI X](https://github.com/mui-org/material-ui-x) is).
+In v4, we have written all the demos TypeScript first.
+
+With v5, we have made new steps toward the adoption of TypeScript:
+
+- We have made the TypeScript definitions the source of the API pages.
+  This reduces the likeliness for a new release to include out-of-date definitions.
+- We have migrated our first component to be written in TypeScript.
+- We are writing most of the new code in TypeScript.
+
+Overall, the language stats of GitHub give some qualitative measure of the progress:
+
+- 02/2019: 1.6%
+- 03/2020: 3.2%
+- 07/2020: 5.3%
+- 10/2020: 25.4%
+- 07/2021: 35.4%
+- Today: 36.1%
+
+<img loading="lazy" src="/static/blog/mui-core-v5/typescript.png" alt="Screenshot of our current language distribution. 36.1% TypeScript, 63.9% JavaScript." style="width: 700px; margin-bottom: 16px;" />
+
+We expect the adoption of TypeScript to increase during the lifecycle of v5.
+With this organic adoption strategy, it might take us two years to close [#15984](https://github.com/mui-org/material-ui/issues/15984).
+
+### Strict Mode support
+
+This release is compatible with `React.StrictMode`.
+This is the result of two years of work on the infrastructure.
+We now run the tests and the documentation in strict mode.
+Thanks for the patience, this has been an old and [frequently requested](https://github.com/mui-org/material-ui/issues/13394) improvement.
+
+Bonus point, we run [React v18](https://github.com/reactwg/react-18) (unreleased) in a nightly build, in strict mode, without any test failures.
+
+> ⚠️ `@mui/styles` is not compatible with strict mode nor concurrent features.
+
+## A new product line: X
+
+<img loading="lazy" src="/static/blog/mui-core-v5/mui-x-blog-hero.png" alt="Mockup of some MUI X components" style="width: 700px; margin-bottom: 16px;" />
+
+We are very excited to introduce a new product line to the MUI family: [**MUI X**](/x/)!
+We have recently released our [first stable version](https://github.com/mui-org/material-ui-x/releases/tag/v4.0.0).
+
+MUI X embodies our initiative to solve the main pain point developers have reported two years in a row during our developer's survey: [2020](/blog/2020-developer-survey-results/#5-how-can-we-improve-material-ui-for-you), [2019](/blog/2019-developer-survey-results/#4-how-can-we-improve-material-ui-for-you).
+We have heard you, you want **more components**.
+
+In practice, there are so many different components that you might need that, our team can't cover it all.
```

Comment:
```suggestion
But, in practice, to cover the many possible components you might need, we would need an even bigger team to do it. 
```

I got what you were trying to say but found it to be a little confusing, maybe now it's a bit clearer?!